src/BackstageItemUpdater.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace GildedRose;
+
+class BackstageItemUpdater implements Updateable
+{
+    private const UPDATER_NAMES = 'Backstage passes to a TAFKAL80ETC concert';
```

Comment:
nie rozumiem tej nazwy zmiennej - tu jest jedna wartość, a nie kilka i to nie są nazwy updatera